### ISSUE TITLE
cannot get the same result

### ISSUE THREAD

**Issue by gnemix** [MaoXu/Joint_Bayesian#1]
After running your script,  it is only 66%.

**Comment by Mahbubul-Alam-ODU** [comment]
I also have the same problem. Got only 66%. Could you figure it out?

**Comment by gnemix** [comment]
yes I figure it out, you should uncomment one line, to minus the mean when perform PCA projection

**Comment by Mahbubul-Alam-ODU** [comment]
Thank you very much for your reply. I also tried to reduce the input dimension from 2000 to 1000 using PCA. It gave me 88.70%.

**Comment by iamhankai** [comment]
where did you get the WDRef dataset? I cann't connect to the website:http://home.ustc.edu.cn/~chendong/JointBayesian/
Can you send me one? my email: kaihana@163.com
Thank you very much.

**Comment by Mahbubul-Alam-ODU** [comment]
WDRef dataset is not publicly available. The authors only released the
features obtained from WDRef (LBP and another one). You have to run JB on
these features.

On Thu, Oct 1, 2015 at 8:42 AM, HanKai notifications@github.com wrote:

> where did you get the WDRef dataset? I cann't connect to the website:
> http://home.ustc.edu.cn/~chendong/JointBayesian/
> Can you send me one? my email: kaihana@163.com
> Thank you very much.
> 
> —
> Reply to this email directly or view it on GitHub
> 
> ## https://github.com/MaoXu/Joint_Bayesian/issues/1#issuecomment-144715253.
> 
> Spam
> https://www.spamtrap.odu.edu/canit/b.php?i=01PnMHPVq&m=4f1199353aa4&t=20151001&c=s
> Not spam
> https://www.spamtrap.odu.edu/canit/b.php?i=01PnMHPVq&m=4f1199353aa4&t=20151001&c=n
> Forget previous vote
> https://www.spamtrap.odu.edu/canit/b.php?i=01PnMHPVq&m=4f1199353aa4&t=20151001&c=f

## 

_Best Regards,_

_Mahbubul Alam_
PhD Student and Research Assistant
Vision Lab, ECE
Frank Batten College of Engineering & Technology
Old Dominion University
Norfolk, VA-23529, USA

**Comment by iamhankai** [comment]
I just want the features obtained from WDRef, but i cann't connect to that website. So can you send me the features data?

**Comment by Mahbubul-Alam-ODU** [comment]
I just checked. this link: http://home.ustc.edu.cn/~chendong/JointBayesian/
is working just fine for me. At the bottom of this page there is one link
called "WDRef database
https://skydrive.live.com/redir?resid=D472C693301B900F!109&authkey=!AHC4sGY3p8_xNAA".
If you click on this it will take you to the database and it is also
working for me.

On Fri, Oct 2, 2015 at 12:12 AM, HanKai notifications@github.com wrote:

> I just want the features obtained from WDRef, but i cann't connect to that
> website. So can you send me the features data?
> 
> —
> Reply to this email directly or view it on GitHub
> 
> ## https://github.com/MaoXu/Joint_Bayesian/issues/1#issuecomment-144912295.
> 
> Spam
> https://www.spamtrap.odu.edu/canit/b.php?i=03Po4cRV2&m=a3d0b03207dc&t=20151002&c=s
> Not spam
> https://www.spamtrap.odu.edu/canit/b.php?i=03Po4cRV2&m=a3d0b03207dc&t=20151002&c=n
> Forget previous vote
> https://www.spamtrap.odu.edu/canit/b.php?i=03Po4cRV2&m=a3d0b03207dc&t=20151002&c=f

## 

_Best Regards,_

_Mahbubul Alam_
PhD Student and Research Assistant
Vision Lab, ECE
Frank Batten College of Engineering & Technology
Old Dominion University
Norfolk, VA-23529, USA

**Comment by Zhangxiaof001** [comment]
Hi , my friend ! This link can't connected . Can you share the WDRef dataset to me ? Look forward to your reply .
